### PR TITLE
feat: docs quickstart

### PR DESCRIPTION
**Pull request by 0xAlec**
**What changed? Why?**
docs

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/getting-started.mdx [hunk 1]
```diff
@@ -42,54 +42,53 @@ npm i @coinbase/onchainkit@latest @wagmi/core@latest
...
+```tsx twoslash [providers.tsx]
 // @noErrors: 2307 2580 - cannot find 'process', cannot find './wagmi'
-'use client';
-import { ReactNode } from 'react';
 import { OnchainKitProvider } from '@coinbase/onchainkit';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { base } from 'viem/chains';
-import { WagmiProvider } from 'wagmi';
-import { wagmiConfig } from './wagmi';
+import { base } from 'wagmi/chains'; // [!code ++]
```
Comment (alessey) [inline site/docs/pages/getting-started.mdx:57]: Should this be imported from viem to match the OnchainKitProviderReact type?

**Review by fakepixels — APPROVED** [review]

File: site/docs/pages/getting-started.mdx (hunk 1 above)
Comment (0xAlec) [reply site/docs/pages/getting-started.mdx:57]: I'm importing in my demo app from `wagmi/chains` with no issue

Comment (0xAlec) [reply site/docs/pages/getting-started.mdx:57]: I think we should standardize on `wagmi` considering we use wagmi in the rest of the files

File: site/docs/pages/getting-started.mdx [hunk 2]
```diff
@@ -138,17 +115,39 @@ import '@coinbase/onchainkit/styles.css';
 
 For example, if you're using Next.js with the app router, your `app/layout.tsx` might look like this:
 
-```tsx
-import '@coinbase/onchainkit/styles.css'; // Add this line
-import { OnchainProviders } from './OnchainProviders'
-
-export default function RootLayout({ children }) {
+```tsx [layout.tsx]
```
Comment (alessey) [inline site/docs/pages/getting-started.mdx:118]: add twoslash

**Review by alessey — APPROVED** [review]

**Pull request merged**